cloud/service-operations/general.md

```diff
@@ -9,6 +9,9 @@ testing purposes, or to prepare for a major version upgrade. The only difference
 between the original and the forked service is that the `tsdbadmin` user has a
 different password.
 
+The fork is created by restoring from backup and applying the Write-Ahead Log.
```

Comment:
Do we capitalise write-ahead log elsewhere?

Reply:
Good question. We seem to be inconsistent. Do you have a preference? I can go through the docs and update everything to one style (capitalized or not)

Reply:
I prefer fewer initcaps, generally. In the backups section, I used lowercase in the index (https://docs.timescale.com/timescaledb/latest/how-to-guides/backup-and-restore/) but then basically referred to WAL-E in the main section. 

![image](https://user-images.githubusercontent.com/3914967/168703610-06c74bbb-8d35-42bc-8ea3-8b1d85594637.png)

I think we can probably ditch the initcaps.